tests/test_noise.py

```diff
@@ -360,3 +361,21 @@ def test_param_noise_apply(
         psi_expected = op_concrete(psi_init, values=values)
         psi_star = op_concrete_noise(psi_init, values=values)
         assert torch.allclose(psi_star, psi_expected, rtol=RTOL, atol=ATOL)
+
+
+def test_analog_noise_add():
```

Comment:
Isn't there some known example of analog noisy simulations that we can compare with ?

Reply:
Hm could not find any, usually this is compared to mesolve which is already tested.